services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -413,33 +412,6 @@ const indexNewBlock = async block => {
 	}
 };
 
-const buildIndex = async (from, to) => {
```

Comment:
Are these legacy code that we not use anymore?

Reply:
Yes, I verified and it is not being used anywhere.